scripts/bin/fks-inityear.sh

```diff
@@ -46,35 +44,13 @@ for file in $(find . -name "*.$seminar" -type f) ; do
 	mv $file ${file%.$seminar}
 done
 
-# create dependency Makefile part
-MAKEFILE="problems/Makefile-manual.inc"
-
-cat >$MAKEFILE <<END
-# Makefile part for problems dependent images
-# Problem TeX file must be first dependecy in order building process worked
-# correctly
-
-END
-
-echo >>$MAKEFILE
-for B in $(seq 1 $2); do
-	echo  >> $MAKEFILE
-	for P in $(seq 1 $1); do
-		echo "problem${B}_$P= problem$B-$P.tex" >>$MAKEFILE
-	done
-done
-
 #SOAP files parameters
-
 for file in $(find . -name "*.soap" -type f) Makefile.inc; do
-	sed -i "s/SEM/$seminar/" 	$file
-	sed -i "s/YEAR/$rocnik/" 	$file
+	sed -i "s/SEM/$seminar/" $file
+	sed -i "s/YEAR/$rocnik/" $file
 done
 
 
-
 echo "Year initialized. Before commiting do not forget to modify contest specific information."
 echo "This mostly applies to:"
 echo "    - contest name in Makefile.conf.sample"
```

Comment:
- contest name in Makefile.conf (create by cp Makefile.conf.sample Makefile.conf)
Předpokládám, že `.sample` neměníme.

Reply:
`Makefile.conf` může obsahovat citlivé informace, aka jméno a heslo do DB. Z tohoto důvodu jej nechceme commitovat a místo toho v repu existuje `Makefile.conf.sample`. Samotný `make` potom v nové verzi kontroluje, jestli `Makefile.conf` existuje a pokud ne, tak ho vykopíruje.
Co se týče změny v samotném souboru, tak tam se jedná o odkomentování řádku, který definuje proměnnou `XSL_RESULTS`, protože FYKOS a Výfuk mají rozdílné soubory. Mimochodem, jestliže tato proměnná není nastavená (tedy jsou zakomentovány oba řádky), tak generování výsledků vyhodí error `... is not a stylesheet`.